diff --git a/node_modules/@apidevtools/json-schema-ref-parser/lib/dereference.js b/node_modules/@apidevtools/json-schema-ref-parser/lib/dereference.js
index 65811e3..7ad234a 100644
--- a/node_modules/@apidevtools/json-schema-ref-parser/lib/dereference.js
+++ b/node_modules/@apidevtools/json-schema-ref-parser/lib/dereference.js
@@ -39,7 +39,7 @@ function crawl (obj, path, pathFromRoot, parents, $refs, options) {
     circular: false
   };
 
-  if (obj && typeof obj === "object") {
+  if (obj && typeof obj === "object" && !ArrayBuffer.isView(obj)) {
     parents.push(obj);
 
     if ($Ref.isAllowed$Ref(obj, options)) {
diff --git a/node_modules/@apidevtools/json-schema-ref-parser/lib/resolve-external.js b/node_modules/@apidevtools/json-schema-ref-parser/lib/resolve-external.js
index dad8255..c7238fb 100644
--- a/node_modules/@apidevtools/json-schema-ref-parser/lib/resolve-external.js
+++ b/node_modules/@apidevtools/json-schema-ref-parser/lib/resolve-external.js
@@ -54,7 +54,7 @@ function resolveExternal (parser, options) {
 function crawl (obj, path, $refs, options) {
   let promises = [];
 
-  if (obj && typeof obj === "object") {
+  if (obj && typeof obj === "object" && !ArrayBuffer.isView(obj)) {
     if ($Ref.isExternal$Ref(obj)) {
       promises.push(resolve$Ref(obj, path, $refs, options));
     }
