diff --git a/node_modules/aframe-environment-component/index.js b/node_modules/aframe-environment-component/index.js
index dbfdff5..62ef434 100644
--- a/node_modules/aframe-environment-component/index.js
+++ b/node_modules/aframe-environment-component/index.js
@@ -254,7 +254,6 @@ AFRAME.registerComponent('environment', {
       Object.assign(this.environmentData, this.data);
       Object.assign(this.environmentData, this.presets[this.data.preset]);
       Object.assign(this.environmentData, this.el.components.environment.attrValue);
-      console.log(this.environmentData);
     }
 
     var skyType = this.environmentData.skyType;
@@ -406,7 +405,6 @@ AFRAME.registerComponent('environment', {
       str += ', ';
     }
     str += '}';
-    console.log(str);
   },
 
   // dumps current component settings to console.
@@ -450,7 +448,6 @@ AFRAME.registerComponent('environment', {
         }
       }
     }
-    console.log('%c' + params.join('; '), 'color: #f48;font-weight:bold');
   },
 
   // Custom Math.random() with seed. Given this.environmentData.seed and x, it always returns the same "random" number
@@ -711,7 +708,6 @@ AFRAME.registerComponent('environment', {
         var diff = new THREE.Color(col2.r - col1.r, col2.g - col1.g, col2.b - col1.b);
         var perlin = new PerlinNoise();
         for (i = 0, j = 0, numpixels = im.length; i < numpixels; i += 4, j++){
-          //console.log( (j % size) / size, j / size)
           var rnd = perlin.noise((j % size) / size * 3, j / size / size * 3, 0);
           im[i + 0] = Math.floor((col1.r + diff.r * rnd) * 255);
           im[i + 1] = Math.floor((col1.g + diff.g * rnd) * 255);
