diff --git a/node_modules/express/lib/view.js b/node_modules/express/lib/view.js
index d66b4a2..6d9a93e 100644
--- a/node_modules/express/lib/view.js
+++ b/node_modules/express/lib/view.js
@@ -78,7 +78,7 @@ function View(name, options) {
     debug('require "%s"', mod)
 
     // default engine export
-    var fn = require(mod).__express
+    var fn = null // require(mod).__express
 
     if (typeof fn !== 'function') {
       throw new Error('Module "' + mod + '" does not provide a view engine.')
